packages/labs/context/src/lib/directives/provide.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  Directive,
+  directive,
+  DirectiveParameters,
+  DirectiveResult,
+  ElementPart,
+  PartInfo,
+  PartType,
+} from 'lit/directive.js';
+import {noChange} from 'lit';
```

Comment:
When forking this experiment, and in now lit 2.0, `noChange` is not found. It is only accessible via `lit-html`.

Reply:
Directives have been removed as they have sequencing issues that are more easily solved via use of decorators and elements.